example/src/App.tsx

```diff
@@ -94,8 +55,8 @@ const App = () => {
             .
           </h1>
           <Importer
-            theme={currentTheme}
-            locale={currentLocale}
+            theme="default"
+            locale="en"
```

Comment:
Aren't those 2 default values? Do we need to pass them? From what I see theme is required, but I guess we could make it optional with default value?